index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+      <div id="main">
```

Comment:
As it mentioned in this awesome [blog](http://diveintohtml5.it/semantics.html), "The div element has no defined semantics, and the id attribute has no defined semantics."

A preferable use in this case would be a header tag as follows:

    <header>
        <h1>YOUR H1</h1>
    </header>

This does not mean that you should close your header where your actual closing div is. Instead of having the main div with multiple nested branches, a better approach is to divide your content into sections.

[Pluralsight](https://www.pluralsight.com/guides/semantic-html) recommends
 
    <header>
      Here goes logo, navigation, etc.
    </header>
    <main>
      A place for website's main content
    </main>
    <footer>
      Footer information, links, etc.
    </footer>

If you wan to go a little bit beyond you could also use _section_ tags to further segment your code better.